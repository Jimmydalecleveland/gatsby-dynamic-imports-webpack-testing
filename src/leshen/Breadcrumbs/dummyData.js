export const CRUMB_DATA = [
  {
    shortName: 'Home',
    path: '/',
  },
  {
    shortName: 'Internet',
    path: '/internet',
  },
  {
    shortName: 'Gigabit',
    path: '/gigabit',
  },
  {
    shortName: 'Gigabit Plans',
    path: '/gigabit-plans',
  },
]
