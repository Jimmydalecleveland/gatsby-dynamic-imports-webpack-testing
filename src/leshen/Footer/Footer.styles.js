import styled from '@emotion/styled'
import Section from '../Section'
import Typography from '../Typography'

export const TopSection = styled(Section)`
  padding-top: ${({ theme }) => theme.spacing.lg} !important;
  padding-bottom: ${({ theme }) => theme.spacing.lg} !important;
  border-bottom: 1px solid
    ${({ theme, dark }) =>
      dark ? theme.colors.dark.lighter.hex : theme.colors.light.dark.hex};

  .logo {
    display: inline-block;
    max-width: 200px;
  }
`

export const BottomSection = styled(Section)`
  display: grid;
  grid-template-columns: repeat(auto-fit, minmax(132px, 1fr));
  grid-row-gap: ${({ theme }) => theme.spacing.md};
  grid-column-gap: ${({ theme }) => theme.spacing.md};

  padding-top: ${({ theme }) => theme.spacing.lg} !important;
  padding-bottom: ${({ theme }) => theme.spacing.lg} !important;
`

export const LegalSection = styled(Section)`
  padding-top: ${({ theme }) => theme.spacing.lg} !important;
  padding-bottom: ${({ theme }) => theme.spacing.lg} !important;
  background-color: ${({ theme, dark }) =>
    dark ? theme.colors.dark.light.hex : theme.colors.light.dark.hex};
`

export const Nav = styled.nav`
  a {
    display: block;
    padding-bottom: 1em;
    text-decoration: none;

    &:hover {
      text-decoration: underline;
    }
  }

  @media (min-width: ${({ theme }) => theme.breakpoints.md}) {
    margin-bottom: 0;
  }
`

export const GroupLabel = styled(Typography)`
  margin-top: 0;
  font-weight: ${({ theme }) => theme.fonts.subheadings.md.weight};
`
