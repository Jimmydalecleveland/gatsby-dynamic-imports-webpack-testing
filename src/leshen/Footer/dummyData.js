export const FOOTER_NAV_ITEMS = [
  {
    text: 'Services',
    href: false,
    subnav: [
      {
        text: 'Clearlink Internet',
        href: '#',
      },
      {
        text: 'FiOS Internet',
        href: '#',
      },
      {
        text: 'Bundles',
        href: '#',
      },
      {
        text: 'Frontier TV',
        href: '#',
      },
      {
        text: 'Dish TV',
        href: '#',
      },
      {
        text: 'Clearlink Phone',
        href: '#',
      },
      {
        text: 'Business',
        href: '#',
      },
    ],
  },
  {
    text: 'About',
    href: false,
    subnav: [
      {
        text: 'Store Locations',
        href: '#',
      },
      {
        text: 'Media Center',
        href: '#',
      },
      {
        text: 'Scholarship',
        href: '#',
      },
      {
        text: 'Contact Us',
        href: '#',
      },
    ],
  },
  {
    text: 'Español',
    href: false,
    subnav: [
      {
        text: 'Internet',
        href: '#',
      },
      {
        text: 'FiOS Internet',
        href: '#',
      },
      {
        text: 'DishLATINO',
        href: '#',
      },
      {
        text: 'Teléfono',
        href: '#',
      },
      {
        text: 'Paquetes',
        href: '#',
      },
    ],
  },
  {
    text: 'Terms',
    href: false,
    subnav: [
      {
        text: 'Legal',
        href: '#',
      },
      {
        text: 'Privacy Policy',
        href: '#',
      },
      {
        text: 'Terms of Service',
        href: '#',
      },
    ],
  },
]

export const DISCLAIMERS = [
  '©2019 Internet Service Partners. All Rights Reserved. The name CenturyLink and the pathways logo are trademarks of CenturyLink.',
  '*Geographic and service restrictions apply to AT&T services. Call to see if you qualify. Fiber optics may apply to all or part of the network, depending on your location.',
  '© 2019 AT&T Intellectual Property. AT&T, the AT&T logo and all other AT&T marks contained herein are trademarks of AT&T Intellectual Property and/or AT&T affiliated companies. All rights reserved. Subsidiaries and affiliates of AT&T Inc. provide products and services under the AT&T brand.',
]
