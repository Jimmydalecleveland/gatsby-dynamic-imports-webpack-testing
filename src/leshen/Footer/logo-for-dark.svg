<?xml version="1.0" encoding="UTF-8"?>
<svg width="155px" height="30px" viewBox="0 0 155 30" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
    <!-- Generator: Sketch 58 (84663) - https://sketch.com -->
    <title>logo</title>
    <desc>Created with Sketch.</desc>
    <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
        <path d="M10.0867347,6.41721975 C13.44848,6.41721975 16.0542347,7.42581645 17.9029021,9.44337542 C18.8281193,10.57334 19.5161958,11.8682587 19.9707922,13.3275683 L20.088987,13.7296372 L14.2048361,13.7296372 C13.9237158,13.1703212 13.6444233,12.7495547 13.3643998,12.4687999 C12.6350955,11.6842952 11.5994469,11.2920429 10.2541639,11.2920429 C9.02147524,11.2920429 8.01287854,11.6842952 7.22873939,12.4687999 C6.21941156,13.5903565 5.71529599,15.439024 5.71529599,18.0155334 C5.71529599,20.5942363 6.21941156,22.4154863 7.22873939,23.4792834 C7.9562158,24.3763825 9.04889269,24.8242009 10.5064045,24.8242009 C11.4020413,24.8242009 12.1693644,24.644708 12.8178785,24.2974202 C13.4100955,23.9753565 13.7548243,23.4727032 14.345579,22.9378801 C14.5467865,22.755682 14.7641375,22.6354839 14.9832197,22.5544511 L15.1476309,22.5006632 L15.2137983,22.4834815 L15.3927429,22.4429951 L15.5683974,22.4151207 L15.6042229,22.411465 C15.7584917,22.394649 15.9061804,22.3873377 16.0407087,22.3873377 L20.0360252,22.3866487 L20.0883425,22.3866387 L20.2556851,22.3866065 C19.6393408,24.7397551 18.4625837,26.5339532 16.726145,27.7655452 C14.9889752,29.0563589 12.8032559,29.699024 10.1704493,29.699024 C6.97649882,29.699024 4.48260731,28.6633754 2.68987146,26.5895193 C0.895673349,24.5719603 3.65566038e-05,21.6869132 3.65566038e-05,17.9321844 C3.65566038e-05,14.3467127 0.868255896,11.5439179 2.60579127,9.52672447 C4.39852712,7.45506174 6.89241863,6.41721975 10.0867347,6.41721975 Z M40.4890342,6.41721975 C43.8507795,6.41721975 46.4565342,7.42581645 48.3052017,9.44337542 C50.0293377,11.4367648 50.9254071,14.18116 50.991785,17.6768857 L50.9957677,18.0999792 L50.9957677,19.0244957 C50.9957677,19.2593719 50.9752046,19.4338281 50.936617,19.5501915 L50.9113219,19.613057 L36.3705672,19.613057 C36.4257677,21.3505924 36.8465342,22.6388471 37.6314045,23.4792834 C38.3588809,24.3763825 39.4511922,24.8242009 40.9090696,24.8242009 C41.8098243,24.8242009 42.5818998,24.6428801 43.2326073,24.2912056 C43.8189752,23.9687764 44.1633384,23.4701443 44.7508031,22.9378801 C45.2929375,22.4465594 45.9553432,22.3873377 46.4466639,22.3873377 L47.5953683,22.3871927 L50.6587158,22.3866065 C50.0423715,24.7397551 48.8656144,26.5339532 47.1291757,27.7655452 C45.3912748,29.0563589 43.2062866,29.699024 40.57348,29.699024 C37.3795295,29.699024 34.8852724,28.6633754 33.0929021,26.5895193 C31.2979729,24.5719603 30.4027017,21.6869132 30.4027017,17.9321844 C30.4027017,14.3467127 31.2712866,11.5439179 33.0084564,9.52672447 C34.8011922,7.45506174 37.2950837,6.41721975 40.4890342,6.41721975 Z M62.550579,6.41721975 C65.6885979,6.41721975 68.042112,6.9505806 69.6107559,8.01401221 C71.1614871,9.08493792 71.9727626,10.5618979 72.043311,12.4483898 L72.0483502,12.7206749 L72.0483502,20.3690476 C72.0483502,22.6114297 71.6411097,24.3617598 70.8295531,25.6225971 C70.0165342,26.8837999 69.0503432,27.7951561 67.9298833,28.3541065 C66.7527606,28.971182 65.5058149,29.3484462 64.1894116,29.4891891 C62.8730083,29.6281042 61.7101427,29.699024 60.7019116,29.699024 C58.683987,29.699024 56.9186686,29.1528683 55.4063219,28.0601915 C53.8932441,26.967149 53.1372535,25.1601561 53.1372535,22.6388471 C53.1372535,20.285333 53.8223243,18.6044603 55.1957559,17.5958636 C55.8614517,17.1067363 56.5947771,16.7137528 57.3602724,16.405215 C58.296487,16.0275853 59.2740106,15.776807 60.2712748,15.6298495 C61.3065578,15.4774084 62.3433031,15.3348377 63.3738337,15.1527858 C63.6458149,15.1045311 63.9170649,15.0493306 64.1846592,14.9806042 C64.79223,14.8252386 65.429046,14.6113825 65.9189045,14.2059698 C66.2665578,13.9179037 66.4990578,13.5227268 66.5008856,13.0569957 C66.5008856,12.2165594 66.0662276,11.6704037 65.1983738,11.4185287 C64.3290578,11.165557 63.5299304,11.0398023 62.8028196,11.0398023 C62.0175837,11.0398023 61.3167936,11.1531278 60.7019116,11.3764886 C60.1458857,11.5781714 59.727254,11.9385684 59.4454837,12.4576795 L59.3566285,12.6365948 L53.7254493,12.6365948 C53.9488101,10.9008872 54.7336804,9.47188957 56.079329,8.3499674 C57.5913101,7.06207825 59.7485153,6.41721975 62.550579,6.41721975 Z M25.3198715,0.365639566 C26.1040106,0.365639566 26.7766521,0.632137208 27.3370649,1.16403579 C27.8448694,1.64857696 28.1236663,2.2364434 28.1701599,2.92845908 L28.1771356,3.13918909 L28.1771356,29.2789886 L22.2940814,29.2789886 L22.2940814,0.365639566 L25.3198715,0.365639566 Z M87.2178785,6.92133532 L87.2178785,12.8888353 L85.2851309,12.8888353 C83.658362,12.8888353 82.4549186,13.2818188 81.6704139,14.0655924 C80.934598,14.8020937 80.5441777,15.930579 80.4982495,17.4540602 L80.4936568,17.764024 L80.4936568,29.2789886 L74.6106026,29.2789886 L74.6106026,18.4362999 C74.6106026,16.2513117 74.9326663,14.4161702 75.5771592,12.931241 C76.2201899,11.4466773 77.103763,10.2556632 78.2242229,9.35892966 C79.2887512,8.51849334 80.5356969,7.90251457 81.9646946,7.50953108 C83.2345898,7.1621621 84.5379907,6.96759071 85.8746405,6.92864112 L86.3774422,6.92133532 L87.2178785,6.92133532 Z M92.8051899,0.365639566 C93.5896946,0.365639566 94.2619705,0.632137208 94.8227488,1.16403579 C95.3318826,1.64857696 95.609592,2.2364434 95.6558769,2.92845908 L95.6628196,3.13918909 L95.6628196,29.2789886 L89.7801309,29.2789886 L89.7801309,0.365639566 L92.8051899,0.365639566 Z M101.922772,6.92133532 C102.707277,6.92133532 103.379187,7.20282117 103.940331,7.76177164 C104.448136,8.27190243 104.727235,8.87357579 104.773784,9.56816499 L104.780768,9.7793306 L104.780768,29.2789886 L98.8977134,29.2789886 L98.8977134,6.92133532 L101.922772,6.92133532 Z M117.197218,6.41721975 C119.661499,6.41721975 121.930933,7.06207825 124.005154,8.3499674 C126.010697,9.59843437 127.046671,12.0787182 127.111421,15.7914814 L127.114659,16.166866 L127.114659,29.2789886 L124.172949,29.2789886 C123.332147,29.2789886 122.645614,29.0132221 122.11335,28.4805924 C121.628809,27.9963836 121.36542,27.385284 121.321536,26.6464698 L121.314954,26.4209933 L121.314954,14.9901089 C121.314954,13.6451915 120.908445,12.66584 120.096888,12.048399 C119.283869,11.4324202 118.37361,11.1242481 117.365013,11.1242481 C116.356051,11.1242481 115.459317,11.4324202 114.674812,12.048399 C113.942267,12.6246773 113.552053,13.5162198 113.50328,14.7259985 L113.498055,14.9901089 L113.498055,29.2789886 L107.615001,29.2789886 L107.615001,16.166866 C107.615001,12.3565712 108.608975,9.80821032 110.598386,8.51849334 C112.586699,7.23023862 114.787407,6.53017966 117.197218,6.41721975 Z M129.760992,0.365639566 L132.786416,0.365639566 C133.570555,0.365639566 134.243562,0.646394283 134.80361,1.20571032 C135.312079,1.71584111 135.591541,2.29485541 135.63815,2.94275324 L135.645143,3.13918909 L135.645143,15.1586348 L142.788303,6.92133532 L149.848846,6.92133532 L141.864152,15.4949556 L149.932926,29.2789886 L142.872383,29.2789886 L138.753551,21.713965 C137.912749,20.2012528 137.221098,19.7165122 136.568928,19.7812174 C135.998645,19.8379083 135.694062,20.267762 135.650577,20.6857792 L135.645143,20.7898141 L135.645143,29.2789886 L129.760992,29.2789886 L129.760992,0.365639566 L132.786416,0.365639566 Z M152.810662,25.1085804 C153.090685,25.1100735 153.354258,25.1630806 153.601015,25.2683636 C153.847407,25.3732811 154.063091,25.5173141 154.247702,25.7000971 C154.432313,25.8828801 154.578539,26.0981985 154.686015,26.3464179 C154.793857,26.5942716 154.84723,26.861866 154.845797,27.1488353 C154.84394,27.4434815 154.788008,27.7150971 154.677607,27.963682 C154.567206,28.2122669 154.418787,28.4279509 154.232348,28.6103683 C154.045544,28.7931514 153.828763,28.934991 153.580544,29.0351561 C153.33269,29.1360523 153.069117,29.1850382 152.789093,29.1839595 C152.508704,29.1824792 152.2444,29.1302032 151.995815,29.0271136 C151.74723,28.924024 151.530084,28.7796254 151.343645,28.5950146 C151.157206,28.4104037 151.010249,28.1936231 150.902407,27.9435759 C150.79493,27.6938943 150.741558,27.4215476 150.742991,27.1272669 C150.744482,26.8395665 150.800779,26.5727032 150.910815,26.3259462 C151.021581,26.0791891 151.170732,25.865333 151.358999,25.6847433 C151.547265,25.5037882 151.766239,25.3619485 152.016287,25.2599556 C152.265603,25.1575971 152.530638,25.107149 152.810662,25.1085804 Z M152.809199,25.408741 C152.569022,25.4072787 152.346027,25.4507811 152.140213,25.5388825 C151.934034,25.6266183 151.755272,25.7476207 151.603563,25.901524 C151.451853,26.0554273 151.332678,26.2374792 151.246039,26.4476797 C151.1594,26.6582457 151.115532,26.8852622 151.11407,27.1287292 C151.112607,27.3802386 151.153916,27.6123731 151.238728,27.8254981 C151.322808,28.0386231 151.440154,28.223965 151.590402,28.3807929 C151.74065,28.5383518 151.917949,28.6619132 152.123032,28.752208 C152.327749,28.8425028 152.550744,28.887833 152.790555,28.8892952 C153.026711,28.8907575 153.247147,28.8476207 153.451133,28.7591537 C153.655485,28.6710523 153.833515,28.5493188 153.985225,28.3935877 C154.136935,28.2382221 154.25611,28.0543424 154.342749,27.8423141 C154.429022,27.6295547 154.473256,27.3981514 154.474739,27.1470075 C154.475815,26.9031749 154.434506,26.6757929 154.35006,26.4644957 C154.265614,26.252833 154.148633,26.0696844 153.998751,25.9139532 C153.848138,25.7585877 153.67157,25.6357575 153.468315,25.5458282 C153.265426,25.4555334 153.045355,25.4098377 152.809199,25.408741 Z M152.009706,25.9501443 L152.931664,25.9548966 C153.215343,25.9563589 153.424812,26.0141183 153.560803,26.1274438 C153.696428,26.2407693 153.763692,26.4122198 153.762596,26.6410641 C153.761499,26.8556514 153.700815,27.0128448 153.580178,27.1122787 C153.4897,27.1868542 153.383183,27.2381934 153.260165,27.2666047 L153.131629,27.2873848 L153.808291,28.3438707 L153.409824,28.3416773 L152.766062,27.3122433 L152.373445,27.3107811 L152.367961,28.3358282 L151.996912,28.3340004 L152.009706,25.9501443 Z M152.378928,26.2521018 L152.374907,27.0102858 L152.762407,27.0124792 C152.846121,27.0128448 152.925084,27.0106514 152.999659,27.0055334 C153.074235,27.0004155 153.140768,26.9854273 153.198893,26.9602032 C153.257383,26.9349792 153.30381,26.8962292 153.338904,26.8435877 C153.373633,26.7913117 153.39118,26.7174674 153.391553,26.6227858 C153.391912,26.5430924 153.376923,26.4791183 153.346216,26.4315948 C153.315508,26.3844367 153.276027,26.3478801 153.227041,26.3219249 C153.17769,26.2963353 153.121758,26.2784226 153.05815,26.2692834 L152.963514,26.2585449 L152.870249,26.2546608 L152.378928,26.2521018 Z M66.5849658,18.0155334 C65.7993644,18.4085169 64.7066875,18.7726207 63.3069351,19.1085759 C63.3069351,19.1085759 62.3623125,19.3176797 61.3738219,19.5289768 C60.3856969,19.7406396 59.5986333,20.1635995 59.1881026,20.6216537 C58.7779375,21.079708 58.5995413,21.8550735 58.5995413,22.4706867 C58.5995413,23.3111231 58.852513,23.9837646 59.3566285,24.4878801 C59.8607441,24.9371608 60.5882205,25.1601561 61.5416167,25.1601561 C63.1102606,25.1601561 64.3433149,24.7127032 65.2407795,23.8152386 C66.080439,22.9762646 66.5271507,21.8397684 66.5797097,20.4075572 L66.5849658,20.1171726 L66.5849658,18.0155334 Z M40.6575601,11.2920429 C39.4241403,11.2920429 38.4148125,11.6842952 37.6314045,12.4687999 C36.8988589,13.202028 36.4834881,14.2263179 36.3864811,15.5440478 L36.3705672,15.8309108 L44.9434564,15.8309108 C44.8312276,14.3174674 44.4393408,13.1977386 43.7663337,12.4687999 C43.0377606,11.6842952 42.0013809,11.2920429 40.6575601,11.2920429 Z M101.922772,0.365639566 C102.707277,0.365639566 103.379187,0.646394283 103.940331,1.20571032 C104.448136,1.71584111 104.727235,2.31811871 104.773784,3.01199381 L104.780768,3.22290372 L104.780768,4.98822211 L98.8977134,4.98822211 L98.8977134,0.365639566 L101.922772,0.365639566 Z" id="logo" fill="#FFFFFF" fill-rule="nonzero"></path>
    </g>
</svg>