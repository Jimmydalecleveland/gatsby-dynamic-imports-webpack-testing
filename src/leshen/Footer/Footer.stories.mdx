import { Story, Meta, Preview, Props, Anchor } from '@storybook/addon-docs/blocks'

import Footer, { ticketID } from '.'
import { FOOTER_NAV_ITEMS, DISCLAIMERS } from './dummyData'
import logoForDark from './logo-for-dark.svg'
import logo from './logo.svg'

import ComponentProgress from '../ComponentProgress'

<Meta title='Components|Footer' component={Footer} decorators={[(story) => <><ComponentProgress ticketID={ticketID} />{story()}</>]} />

# Footer

<Preview>
  <Story name="default">
    <Footer
      logo={logo}
      navData={FOOTER_NAV_ITEMS}
      disclaimers={DISCLAIMERS}
      siteName="Leshen"
    />
  </Story>
</Preview>

## Props
<Props of={Footer} />

<Preview>
  <Story name="dark">
    <Footer
      dark
      logo={logoForDark}
      navData={FOOTER_NAV_ITEMS}
      disclaimers={DISCLAIMERS}
      siteName="Leshen"
    />
  </Story>
</Preview>

