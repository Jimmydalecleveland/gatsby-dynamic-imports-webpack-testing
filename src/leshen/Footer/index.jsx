import React from "react"
import PropTypes from "prop-types"
import uuid from "uuid/v4"

import Link from "../Link"
import Typography from "../Typography"
import * as Styled from "./Footer.styles"
import { FOOTER } from "../utils/dataTrackingValues"

const Footer = ({ dark, logo, navData, disclaimers }) => (
  <footer>
    {logo && (
      <Styled.TopSection backgroundColor={dark && "Dark"}>
        {logo}
      </Styled.TopSection>
    )}
    {navData && (
      <Styled.BottomSection backgroundColor={dark && "Dark"}>
        {navData.map(({ text, href, subnav }) => (
          <Styled.Nav key={uuid()}>
            <Styled.GroupLabel variant="feature">
              {href ? <Link to={href}>{text}</Link> : text}
            </Styled.GroupLabel>
            {subnav.map(({ href: subNavHref, text: subNavText }) => (
              <Link
                key={uuid()}
                to={subNavHref}
                data-tracking={FOOTER.NAVIGATION_ITEM}
              >
                {subNavText}
              </Link>
            ))}
          </Styled.Nav>
        ))}
      </Styled.BottomSection>
    )}
    {disclaimers && (
      <Styled.LegalSection backgroundColor={dark && "Dark"}>
        {disclaimers.map(disclaimer => (
          <Typography key={uuid()} variant="legal">
            {disclaimer}
          </Typography>
        ))}
      </Styled.LegalSection>
    )}
  </footer>
)

Footer.defaultProps = {
  dark: false,
  logo: null,
  disclaimers: [],
}

Footer.propTypes = {
  dark: PropTypes.bool,
  logo: PropTypes.node,
  navData: PropTypes.arrayOf(
    PropTypes.shape({
      text: PropTypes.string,
      href: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
      subnav: PropTypes.arrayOf(
        PropTypes.shape({
          text: PropTypes.string,
          href: PropTypes.string,
        })
      ),
    })
  ).isRequired,
  disclaimers: PropTypes.arrayOf(PropTypes.string),
}

export default Footer

export const ticketID = "CMS-162"
