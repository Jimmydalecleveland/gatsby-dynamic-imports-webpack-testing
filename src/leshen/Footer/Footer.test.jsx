import React from 'react'
import { render } from 'setup-tests'

import Footer from '.'

import { FOOTER_NAV_ITEMS, DISCLAIMERS } from './dummyData'

describe('Footer', () => {
  test('Footer should render', () => {
    const { container } = render(
      <Footer
        logo={<div>I am a logo</div>}
        navData={FOOTER_NAV_ITEMS}
        disclaimers={DISCLAIMERS}
      />
    )

    expect(container.firstChild).toMatchSnapshot()
  })
})
