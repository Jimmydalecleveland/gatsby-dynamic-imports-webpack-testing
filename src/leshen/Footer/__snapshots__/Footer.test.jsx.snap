// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Footer should render 1`] = `
.emotion-1 {
  padding-left: calc(2.2222222222222223vw + 21.333333333333332px);
  padding-right: calc(2.2222222222222223vw + 21.333333333333332px);
  padding-top: calc(7.777777777777778vw + -5.333333333333333px);
  padding-bottom: calc(7.777777777777778vw + -5.333333333333333px);
  position: relative;
  padding-top: 32px !important;
  padding-bottom: 32px !important;
  border-bottom: 1px solid #e6e8eb;
}

@media (max-width:480px) {
  .emotion-1 {
    padding-left: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-1 {
    padding-left: 48px;
  }
}

@media (max-width:480px) {
  .emotion-1 {
    padding-right: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-1 {
    padding-right: 48px;
  }
}

@media (min-width:1288px) {
  .emotion-1 {
    padding-right: calc((100% - 1200px) / 2);
    padding-left: calc((100% - 1200px) / 2);
  }
}

@media (max-width:480px) {
  .emotion-1 {
    padding-top: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-1 {
    padding-top: 88px;
  }
}

@media (max-width:480px) {
  .emotion-1 {
    padding-bottom: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-1 {
    padding-bottom: 88px;
  }
}

.emotion-1 .logo {
  display: inline-block;
  max-width: 200px;
}

.emotion-24 {
  padding-left: calc(2.2222222222222223vw + 21.333333333333332px);
  padding-right: calc(2.2222222222222223vw + 21.333333333333332px);
  padding-top: calc(7.777777777777778vw + -5.333333333333333px);
  padding-bottom: calc(7.777777777777778vw + -5.333333333333333px);
  position: relative;
  display: grid;
  grid-template-columns: repeat(auto-fit,minmax(132px,1fr));
  grid-row-gap: 24px;
  grid-column-gap: 24px;
  padding-top: 32px !important;
  padding-bottom: 32px !important;
}

@media (max-width:480px) {
  .emotion-24 {
    padding-left: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-24 {
    padding-left: 48px;
  }
}

@media (max-width:480px) {
  .emotion-24 {
    padding-right: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-24 {
    padding-right: 48px;
  }
}

@media (min-width:1288px) {
  .emotion-24 {
    padding-right: calc((100% - 1200px) / 2);
    padding-left: calc((100% - 1200px) / 2);
  }
}

@media (max-width:480px) {
  .emotion-24 {
    padding-top: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-24 {
    padding-top: 88px;
  }
}

@media (max-width:480px) {
  .emotion-24 {
    padding-bottom: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-24 {
    padding-bottom: 88px;
  }
}

.emotion-6 a {
  display: block;
  padding-bottom: 1em;
  -webkit-text-decoration: none;
  text-decoration: none;
}

.emotion-6 a:hover {
  -webkit-text-decoration: underline;
  text-decoration: underline;
}

@media (min-width:768px) {
  .emotion-6 {
    margin-bottom: 0;
  }
}

.emotion-4 {
  font-size: 1.125rem;
  line-height: 1.5rem;
  margin-top: 0;
  font-weight: 500;
}

@media all and (-ms-high-contrast:none),(-ms-high-contrast:active) {
  .emotion-4 {
    width: 100%;
  }
}

.emotion-33 {
  padding-left: calc(2.2222222222222223vw + 21.333333333333332px);
  padding-right: calc(2.2222222222222223vw + 21.333333333333332px);
  padding-top: calc(7.777777777777778vw + -5.333333333333333px);
  padding-bottom: calc(7.777777777777778vw + -5.333333333333333px);
  position: relative;
  padding-top: 32px !important;
  padding-bottom: 32px !important;
  background-color: #e6e8eb;
}

@media (max-width:480px) {
  .emotion-33 {
    padding-left: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-33 {
    padding-left: 48px;
  }
}

@media (max-width:480px) {
  .emotion-33 {
    padding-right: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-33 {
    padding-right: 48px;
  }
}

@media (min-width:1288px) {
  .emotion-33 {
    padding-right: calc((100% - 1200px) / 2);
    padding-left: calc((100% - 1200px) / 2);
  }
}

@media (max-width:480px) {
  .emotion-33 {
    padding-top: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-33 {
    padding-top: 88px;
  }
}

@media (max-width:480px) {
  .emotion-33 {
    padding-bottom: 32px;
  }
}

@media (min-width:1200px) {
  .emotion-33 {
    padding-bottom: 88px;
  }
}

.emotion-26 {
  font-size: 0.625rem;
  line-height: 0.75rem;
  opacity: 0.8;
}

@media all and (-ms-high-contrast:none),(-ms-high-contrast:active) {
  .emotion-26 {
    width: 100%;
  }
}

<footer>
  <section
    class="emotion-0 emotion-1 emotion-2"
  >
    <div>
      I am a logo
    </div>
  </section>
  <section
    class="emotion-23 emotion-24 emotion-2"
  >
    <nav
      class="emotion-6 emotion-7"
    >
      <p
        class="emotion-3 emotion-4 emotion-5"
      >
        Services
      </p>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Clearlink Internet
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        FiOS Internet
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Bundles
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Frontier TV
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Dish TV
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Clearlink Phone
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Business
      </a>
    </nav>
    <nav
      class="emotion-6 emotion-7"
    >
      <p
        class="emotion-3 emotion-4 emotion-5"
      >
        About
      </p>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Store Locations
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Media Center
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Scholarship
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Contact Us
      </a>
    </nav>
    <nav
      class="emotion-6 emotion-7"
    >
      <p
        class="emotion-3 emotion-4 emotion-5"
      >
        Español
      </p>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Internet
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        FiOS Internet
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        DishLATINO
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Teléfono
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Paquetes
      </a>
    </nav>
    <nav
      class="emotion-6 emotion-7"
    >
      <p
        class="emotion-3 emotion-4 emotion-5"
      >
        Terms
      </p>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Legal
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Privacy Policy
      </a>
      <a
        data-tracking="footer__navigation-item"
        href="#"
        rel="noopener noreferrer"
        target="_blank"
      >
        Terms of Service
      </a>
    </nav>
  </section>
  <section
    class="emotion-32 emotion-33 emotion-2"
  >
    <p
      class="emotion-26 emotion-5"
    >
      ©2019 Internet Service Partners. All Rights Reserved. The name CenturyLink and the pathways logo are trademarks of CenturyLink.
    </p>
    <p
      class="emotion-26 emotion-5"
    >
      *Geographic and service restrictions apply to AT&T services. Call to see if you qualify. Fiber optics may apply to all or part of the network, depending on your location.
    </p>
    <p
      class="emotion-26 emotion-5"
    >
      © 2019 AT&T Intellectual Property. AT&T, the AT&T logo and all other AT&T marks contained herein are trademarks of AT&T Intellectual Property and/or AT&T affiliated companies. All rights reserved. Subsidiaries and affiliates of AT&T Inc. provide products and services under the AT&T brand.
    </p>
  </section>
</footer>
`;
